src/mlj_flux.jl

```diff
@@ -426,7 +422,7 @@ function MLJFlux.fit!(
 
     #return  cache, report
 
-    return (fitresult, report, cache)
+    return (fitresult, cache, report)
```

Comment:
@pat-alt @MojiFarmanbar  i have changed the position as we said. this have a negative effect during the testing since now i get (training_losses = (),)  when i use the same order  in mlj_flux_interfacing ( fitresult, cache, _report = MLJBase.fit(model, 0, X, y)) .  the order seems correct ( https://github.com/FluxML/MLJFlux.jl/blob/07e01b7f41bc3dd870dd08c7c97114cc45f91f5e/src/mlj_model_interface.jl#L149)

Reply:
@Rockdeldiablo , then the issue was the order of arguments, right?

Reply:
the content does not correspond.  cache is not empty despite i didn't use it at all. in third position i pass cache as () but i get the training loss. in second position i pass the training loss through report, but i get  a list of numbers that looks like weights. it mess with the update function that does not work as it should

Reply:
@Rockdeldiablo, did you check your environment is correctly set? i suppose your package is in dev mode that you could see your changes immediately?

Reply:
> @Rockdeldiablo, did you check your environment is correctly set? i suppose your package is in dev mode that you could see your changes immediately?

yes. i have updated the env, moved everything to the new api  and now the issue with report is solved.  there is however another issue in the classification task with the history of the loss that grows instead of going down

Reply:
@MojiFarmanbar i think it's better if i close this pull request since it's a mess and perhaps open a different one

Reply:
@Rockdeldiablo, maybe we can keep this PR still open and you can create a new PR. I understand that you might do a lot of experimentation but make sure you don't loose the track of them, try to isolate your experiments from each other otherwise they might collide and get confusing. 
from above, what I would learn: 
- we need to make sure the environment is updated with the changes in the code otherwise we get inconsistent results. 
- Isolating the experiments
do you agree with me?

Reply:
> @Rockdeldiablo, maybe we can keep this PR still open and you can create a new PR. I understand that you might do a lot of experimentation but make sure you don't loose the track of them, try to isolate your experiments from each other otherwise they might collide and get confusing. from above, what I would learn:
> 
> * we need to make sure the environment is updated with the changes in the code otherwise we get inconsistent results.
> * Isolating the experiments
>   do you agree with me?

yes, but i do experiments because it's not clear how should i proceed with mljflux.update. it needs the old_history to work and the only way to pass it is through cache, but patrick told me to not use it. Maybe some code reviews with some code tips could help me find the errors quickly and move ahead. besides, the original project was about adding conformalized bayes support to conformalprediction.jl , not how to merge 2 libraries that i have never used before.

Reply:
I'll take a look at this tomorrow morning

Reply:
> besides, the original project was about adding conformalized bayes support to conformalprediction.jl , not how to add to mljbase support to laplaceredux.

You're free to use any other library for Bayesian models, it doesn't need to be LaplaceRedux. But ConformalPrediction interfaces MLJ, that's a conscious design choice, so support for adding conformalized Bayes was always going to involve working with MLJ.